src/scripts/templates/standalone/Traverse sites tree.js

```diff
@@ -3,6 +3,10 @@
 // Standalone scripts have no template.
 // They are only evaluated when you run them. 
 
+// The following handles differences in printing between Java 7's Rhino JS engine
+// and Java 8's Nashorn JS engine
+if (typeof println == 'undefined') this.println = print;
+
```

Comment:
The script could be changed to remove the usage of print (which would add the newline in Java 8, making the output look wrong) ?